rts/Rml/Backends/RmlUi_Backend.cpp

```diff
@@ -334,36 +334,43 @@ void RmlGui::PresentFrame()
 */
 bool RmlGui::ProcessMouseEvent(const SDL_Event &event)
 {
-	SDL_Event ev(event);
 	if (data == NULL || data->contexts.size() < 1 || !data->initialized)
 	{
 		return false;
 	}
-	switch (ev.type)
-	{
-	case SDL_MOUSEMOTION:
-	case SDL_MOUSEBUTTONDOWN:
-	case SDL_MOUSEBUTTONUP:
-	case SDL_MOUSEWHEEL:
+	bool result = false;
+	for (auto &context : data->contexts)
 	{
```

Comment:
ditto (consider a function that handles 1 context, like it was before, and then call it in the loop)